.travis.yml

```diff
@@ -106,19 +106,19 @@ before_install:
 install:
     - cd $girder_path
     - python setup.py develop
-    - pip install --upgrade 'git+https://github.com/cdeepakroy/ctk-cli'
-    - pip install -r $main_path/requirements.txt
+    - cd $large_image_path
+    - pip install -U -r requirements.txt
+    - pip install -e .
```

Comment:
And, we probably want to do this after the installation of openslide, Pillow, and libtiff, since they rely on our built libraries and that way we don't have to uninstall and reinstall them.